assets/scripts/datalayer/common.js

```diff
@@ -192,7 +192,9 @@ document.shopAnalytics = {
     .on('click', '.products .product a', onProductClick)
     .on('click', '.single_add_to_cart_button', onProductAddToCart)
     .on('click', '.remove_from_cart_button, .woocommerce-cart-form .product-remove > a, .cart_item td.product-remove .remove', onRemoveSingleProduct)
-    .on('click', 'th.product-remove .remove', onEmptyCart);
+    .on('click', 'th.product-remove .remove', onEmptyCart)
```

Comment:
Hm… can we be less specific with the selector? Ideally we are trying to use only classes, not tags like `th` to ensure future flexibility with the markup.

Reply:
I did not add this 🙈 I move the `;` to the next line so we can add new lines without the need to change two lines again.